frontend/packages/topology/src/components/list-view/TopologyListViewUnassignedGroup.tsx

```diff
@@ -35,6 +35,7 @@ const TopologyListViewUnassignedGroup: React.FC<TopologyListViewUnassignedGroupP
     <DataList aria-label="unassigned items" id="unassigned-items">
       {kindKeys.map((key) => (
         <TopologyListViewKindGroup
+          groupLabel="unassigned"
```

Comment:
We should translate this value when it is displayed or used as an aria-label to the user, or?

```suggestion
          groupLabel={t('topology~unassigned')}
```